Bio/Nexus/Nexus.py

```diff
@@ -1028,6 +1043,7 @@ def _matrix(self, options):
             "Please Report this as a bug, and send in data file."
 
     def _translate(self, options):
+        """Translates a Nexus file (PRIVATE).""""
```

Comment:
There are four double-quotes at the end of that line, which is where this flake8 error comes from:

```
$ flake8 Bio/
Bio/Nexus/Nexus.py:1046:49: E999 SyntaxError: EOL while scanning string literal
...
```